static/js/settings_org.js

```diff
@@ -791,6 +791,12 @@ exports.build_page = function () {
         }
     });
 
+    $("#id_org_profile_preview").click(function (e) {
+        e.stopPropagation();
+        e.preventDefault();
+        window.open('/login/?preview=True', '_blank');
```

Comment:
It'd be cleaner to just have use an `a` tag with the `_blank` tag on it around the button without any JS click handler; we don't use `window.open` anywhere in Zulip, because it's unnecessary.

Reply:
Also let's use `/login/?preview=true`; that's the case used for booleans in JS code.